contracts/Manager.sol

```diff
@@ -731,6 +738,24 @@ contract Manager is Errors, UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Pa
         account.scheduleTransfer(fromGroups, fromVotes, toGroups, toVotes);
     }
 
+    /**
+     * @notice Allows strategy to change strategy for account.
+     * address(0) = default strategy
+     * !address(0) = voting for validator group.
+     * @param account The account to change strategy for.
+     * @param newStrategy The new strategy.
+     */
+    function _changeStrategy(address account, address newStrategy) private {
+        uint256 stCeloAmount = stakedCelo.balanceOf(account) +
+            stakedCelo.lockedVoteBalanceOf(account);
+        if (stCeloAmount != 0) {
+            _transfer(strategies[account], newStrategy, stCeloAmount);
+        }
+
+        strategies[account] = newStrategy;
```

Comment:
Although I agree with this approach in case we have accounting error or some other issue it might be beneficial to be able to revert to account's original strategy - I would hope that multisig will not let these kind of things happen